Documentation/interacting_v3.md

```diff
@@ -180,4 +180,64 @@ compacted revision 5
 # any revisions before the compacted one are not accessible
 $ etcdctl get --rev=4 foo
 Error:  rpc error: code = 11 desc = etcdserver: storage: required revision has been compacted
+```
+
+## Grant leases
+
+Applicants can grant leases from an etcd cluster. Applications can also attach keys to leases. Each lease has a time-to-live (TTL) value. Applications specify the wanted TTL while granting the lease. The actual TTL value is determined by the etcd cluster and it will be at least as large as the one specified by the application. Once the timespan of TTL has elasped, the lease will be revoked. The key attached to the revoked lease are deleted.
```

Comment:
Applications can grant leases for keys from an etcd cluster. When a key is attached to a lease, its lifetime is bound to the lease's lifetime which in turn is governed by a time-to-live (TTL). Each lease has a minimum time-to-live (TTL) value specified by the application at grant time. The lease's actual TTL value is at least the minimum TTL and is chosen by the etcd cluster. Once a lease's TTL elapses, the lease expires and all attached keys are deleted.